geomstats/hypersphere.py

```diff
@@ -134,6 +134,25 @@ def random_uniform(self, n_samples=1):
         point = self.intrinsic_to_extrinsic_coords(point)
 
         return point
+      
+    def random_vonMisesFisher(self, K=10, n_samples=1):
```

Comment:
function name: `random_von_mishes_fisher` to respect python style guidelines:

https://www.python.org/dev/peps/pep-0008/#function-and-variable-names